polynote-frontend/polynote/ui.js

```diff
@@ -929,7 +929,8 @@ export class NotebookUI extends UIEventTarget {
         });
 
         this.cellUI.addEventListener('RunToCursor', () => {
-            const activeCellIdx = Cell.currentFocus.id;
+            const allCellIds = this.cellUI.getCodeCellIds();
+            const activeCellIdx = allCellIds.indexOf(Cell.currentFocus.id);
```

Comment:
Note: cell ID is not necessarily the cell index. It's kind of confusing now that we are using numeric IDs.

Reply:
Ah, good catch, thanks! 😅